test/Conversion/ExportVerilog/sv-dialect.mlir

```diff
@@ -1034,6 +1050,13 @@ hw.module @bindInMod() {
 // CHECK-NEXT:   ._k (a1__k)
 // CHECK-NEXT: //._z (z)
 // CHECK-NEXT: );
+// CHECK-NEXT:  bind remoteInstDut extInst2 signed_2 (
+// CHECK-NEXT:    .signed (signed_0),
```

Comment:
Is this correct? I think `extInst2` will also have it's port "signed" renamed to something which is not a Verilog keyword. 🤔

Reply:
Can we rename extmodule names, given that we can't change their actual external definitions?

Reply:
Right, the extmodule doesnot get renamed. 
I updated the test case to module, and it gets properly renamed.

Reply:
Fair, we shouldn't be renaming external modules. (If we see an external module with a Verilog keyword collision, I guess we throw an error?) However, I think the test was actually weird in that we _don't_ generate external modules from extract test code, but actual modules.

I was nervous that there was _another_ problem with renaming that we didn't handle. However, it was just external modules, rightly, not being renamed. The later commit that shows the renaming happening looks great.

Reply:
Ah good thinking, I overlooked that.